multimodal/setup.py

```diff
@@ -27,14 +27,14 @@
     "Pillow",  # version range defined in `core/_setup_utils.py`
     "tqdm",  # version range defined in `core/_setup_utils.py`
     "boto3",  # version range defined in `core/_setup_utils.py`
+    "torch",  # version range defined in `core/_setup_utils.py`
     "requests>=2.21,<3",
     "jsonschema>=4.14,<4.18",
     "seqeval>=1.2.2,<1.3.0",
     "evaluate>=0.2.2,<0.4.0",
     "accelerate>=0.9,<0.17",
     "timm>=0.9.2,<0.10.0",
-    "torch>=1.9,<1.14",
-    "torchvision<0.15.0",
+    "torchvision>=0.15.0,<0.16.0",
```

Comment:
fyi `torchvision==0.15.0` pins `torch==2.0.0` so technically this drops support for torch v1

Reply:
Thanks for pointing it out. I will loose the lower bound to 0.14 in future PRs.